src/com/dotcms/plugin/saml/v3/rest/api/v1/IdpConfigWriterReader.java

```diff
@@ -0,0 +1,129 @@
+package com.dotcms.plugin.saml.v3.rest.api.v1;
+
+import com.dotmarketing.util.json.JSONArray;
+import com.dotmarketing.util.json.JSONException;
+import com.dotmarketing.util.json.JSONObject;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.*;
+
+public class IdpConfigWriterReader {
+
+    public static final String IDP_CONFIGS = "samlConfigs";
+    public static final String DEFAULT_SAML_CONFIG = "defaultSamlConfig";
+    public static final String DISABLE_SAML_SITES = "disabledSamlSites";
+
+    public static File write(List<IdpConfig> idpConfigList, String defaultIdpConfigId, Map<String, String> disabledSitesMap, String idpConfigPath) throws IOException, JSONException {
+        JSONArray jsonArray = new JSONArray();
+        for (IdpConfig idpConfig : idpConfigList) {
+            final JSONObject joIdp = IdpJsonTransformer.idpToJson(idpConfig);
+            final JSONObject joOnlyId = new JSONObject().put(idpConfig.getId(), joIdp);
+            jsonArray.add(joOnlyId);
+        }
+
+        JSONObject jo = new JSONObject();
+        jo.put(DEFAULT_SAML_CONFIG, defaultIdpConfigId);
+        jo.put(IDP_CONFIGS, jsonArray);
+        jo.put(DISABLE_SAML_SITES, SiteJsonTransformer.getJsonObjecFromtMap(disabledSitesMap));
+
+        File idpConfigFile = new File(idpConfigPath);
+        if (!idpConfigFile.exists()) {
```

Comment:
can write

Reply:
Again, in this case I am just checking if exists in order to create it.